### PR TITLE
Updates to the serve UI

### PR DESCRIPTION
**Pull request by jacobhaven**
Previously un-merged changes to the serve UI (deployed at https://cfssl.org)

### CODE REVIEW

File: cli/serve/serve.go [hunk 1]
```diff
@@ -112,23 +128,13 @@ var v1Endpoints = map[string]func() (http.Handler, error){
...
-			rice.LocateFS,
-			rice.LocateWorkingDirectory,
-		}
-		c := &rice.Config{LocateOrder: order}
-		box, err := c.FindBox(staticDir)
-		if err != nil {
-			return nil, err
-		}
-		return http.FileServer(box.HTTPBox()), nil
+		return &box{rice.MustFindBox("static")}, nil
```
Comment (jacobhaven) [inline cli/serve/serve.go]: This is breaking this test: https://github.com/cloudflare/cfssl/blob/serve-ui-updates/cli/serve/serve_test.go#L41

Comment (lziest) [inline cli/serve/serve.go]: also info, authsign endpoints.

**Comment by jacobhaven** [comment]
Updated to cleanup static box use and fix failing tests.

**Comment by lziest** [comment]
LGTM

**Pull request merged**